data.sql

```diff
@@ -7,3 +7,13 @@ VALUES
     (3, 'Pikachu', '2021-01-07', 1, false, 15.04),
     (4, 'Devimon', '2017-05-12', 5, true, 11);
 
+-- Inserting additional data into the animals table
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES
+    (5, 'Charmander', '2020-02-08', 0, false, -11, NULL),
+    (6, 'Plantmon', '2021-11-15', 2, true, -5.7, NULL),
+    (7, 'Squirtle', '1993-04-02', 3, false, -12.13, NULL),
+    (8, 'Angemon', '2005-06-12', 1, true, -45, NULL),
+    (9, 'Boarmon', '2005-06-07', 7, true, 20.4, NULL),
+    (10, 'Blossom', '1998-10-13', 3, true, 17, NULL),
+    (11, 'Ditto', '2022-05-14', 4, true, 22, NULL);
```

Comment:
Hi, thank you for your submission. I have one required comment on your code:

- **Required**: You added a species column to the INSERT INTO statements, which was not part of the instructions. The instructions only asked for name, date_of_birth, escape_attempts, neutered, and weight_kg columns. Please remove the species column from your code and resubmit.


Please make the required change and resubmit your code for another review. If you have any questions or need any help, feel free to reach out to me. Thank you for your hard work and cooperation.